activestorage/test/controllers/representations/redirect_controller_test.rb

```diff
@@ -132,3 +132,57 @@ class ActiveStorage::Representations::RedirectControllerWithPreviewsWithStrictLo
     assert_equal 100, image.height
   end
 end
+
+class ActiveStorage::Representations::RedirectControllerWithOpenRedirectTest < ActionDispatch::IntegrationTest
+  test "showing existing variant stored in s3" do
+    with_raise_on_open_redirects do
+      blob = create_file_blob filename: "racecar.jpg", service_name: :s3
+
+      get rails_blob_representation_url(
+            filename: blob.filename,
+            signed_blob_id: blob.signed_id,
+            variation_key: ActiveStorage::Variation.encode(resize_to_limit: [100, 100]))
+
+      assert_redirected_to(/racecar\.jpg/)
+    end
+  end
+
+  test "showing existing variant stored in azure" do
+    with_raise_on_open_redirects do
+      blob = create_file_blob filename: "racecar.jpg", service_name: :azure
+
+      get rails_blob_representation_url(
+            filename: blob.filename,
+            signed_blob_id: blob.signed_id,
+            variation_key: ActiveStorage::Variation.encode(resize_to_limit: [100, 100]))
+
+      assert_redirected_to(/racecar\.jpg/)
+    end
+  end
+
+  test "showing existing variant stored in gcs" do
+    with_raise_on_open_redirects do
+      blob = create_file_blob filename: "racecar.jpg", service_name: :gcs
+
+      get rails_blob_representation_url(
+            filename: blob.filename,
+            signed_blob_id: blob.signed_id,
+            variation_key: ActiveStorage::Variation.encode(resize_to_limit: [100, 100]))
+
+      assert_redirected_to(/racecar\.jpg/)
+    end
+  end
+
+  private
+    def with_raise_on_open_redirects
```

Comment:
could you put this method here https://github.com/rails/rails/blob/main/activestorage/test/test_helper.rb

rather than duplicating it in both tests